library/std/src/sys/windows/fs.rs

```diff
@@ -397,6 +397,16 @@ impl File {
                     attr.reparse_tag = buf.ReparseTag;
                 }
             }
+            let mut info: c::FILE_ID_INFO = mem::zeroed();
+            let size = mem::size_of_val(&info);
+            cvt(c::GetFileInformationByHandleEx(
+                self.handle.raw(),
```

Comment:
Given [I/O safety](https://github.com/rust-lang/rust/pull/87329) has been merged, `as_raw_handle` should be used instead of `raw`.